apple/RNCWebView.m

```diff
@@ -311,9 +311,7 @@ - (void)didMoveToWindow
     [_webView addObserver:self forKeyPath:@"estimatedProgress" options:NSKeyValueObservingOptionOld | NSKeyValueObservingOptionNew context:nil];
     _webView.allowsBackForwardNavigationGestures = _allowsBackForwardNavigationGestures;
 
-    if (_userAgent) {
-      _webView.customUserAgent = _userAgent;
-    }
+    _webView.customUserAgent = _userAgent;
```

Comment:
I'm not entirely clear on the lifecycle of this class and if didMoveToWindow could theoretically occur more than once. But it's perfectly valid to set customUserAgent to nil.